app/views/tags/index.html.erb

```diff
@@ -47,7 +47,7 @@
     </div>
   <% else %>
     <div class="grid gap-3 m:gap-4 l:gap-6 m:grid-cols-2 l:grid-cols-3 px-3 m:px-0 pb-3" data-follow-button-container="true">
-      <% @tags.each do |tag| %>
+      <% @tags.includes([:badge]).each do |tag| %>
```

Comment:
I'm not sure what this change is for?

Reply:
Its to address a bullet warning because we are loading badges to show on a tag card.